.github/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+### Description
+
+Please explain the changes you made here.
+
+### Pull request type
+
+- [ ] Bugfix
+- [ ] Algorithm/Module
+- [ ] Feature
+- [ ] Code style update (formatting, renaming)
+- [ ] Refactoring (no functional changes, no api changes)
+- [ ] Build related changes
+- [ ] Documentation content changes
+- [ ] Other (please describe):
+
+### Related issues
+
+Delete if this PR doesn't resolve any issues. Link the issue if it does.
+
+######################################
+
+### Reviewer checklist (this part checks the reviewer)
```

Comment:
The reviewer checks this part?

Reply:
Yes, I thought of it as a list that shows to the reviewer whether the code is ready to be merged with `main` or not. On the other side, a developer does not need to take care of it (not applies to employees :sweat_smile: )

Reply:
Sorry for not being clear, I was trying to say that the word order was the wrong way around